### PR TITLE
Added respondAll method.

### PR DESCRIPTION
**Pull request by serbanghita**
Conversation at https://github.com/sinonjs/nise/issues/71

TODO: More tests

### CODE REVIEW

File: lib/fake-server/index.js [hunk 1]
```diff
@@ -238,6 +238,20 @@ var fakeServer = {
         });
     },
 
+    respondAll: function respondAll() {
+        if (this.respondImmediately) {
+            return;
+        }
+
+        this.queue = this.requests.slice(0);
+        var self = this;
```
Comment (fatso83) [inline lib/fake-server/index.js]: Not needed. The 'this' reference is always in the same context, right?

File: lib/fake-server/index.js [hunk 2]
```diff
@@ -238,6 +238,20 @@ var fakeServer = {
...
+
+        var request;
+        while ((request = this.queue.shift())) {
+            self.processRequest(request);
```
Comment (fatso83) [inline lib/fake-server/index.js]: this.processReq...

**Comment by mroderick** [comment]
@serbanghita could you find the time to make the few requested changes for this PR? Alternatively, would you be fine with someone else taking it across the finish line?

**Comment by mroderick** [comment]
OK, the original author doesn't respond. Would anyone else like to take this across the finish line?

**Comment by serbanghita** [comment]
I'm finishing this tomorrow! Thank you for the reminder!

**Comment by serbanghita** [comment]
@fatso83 comments were addressed, I feel terrible for letting this for so long pending 😓

**Comment by fatso83** [comment]
Thank you!

**Pull request merged**